core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
@dsiddarth I have created hooks in windowDimensionsService in #1591 .You can use it in the following way
` $scope.isNarrow = function() { 
    var screenWidth = windowDimensionsService.getWidth();
    if (screenWidth <= 1280) {
      $scope.isSmallerWindow = true;
    } else { 
      $scope.isSmallerWindow = false;
    } 
   $scope.$apply();
 });
windowDimensionsService.addOnResizeHook($scope.isNarrow);
`